### DISCUSSION TITLE
Solver setup goes to infinite calculation at 42 seconds no matter the drive cycle

### DISCUSSION THREAD

**Discussion by cyasing** [pybamm-team/PyBaMM#2688] [Q&A]
I wrote a program to convert power to current at every time step using simple ohm's law for a battery pack with resistances in terminals and busbar with PyBaMM. I simulate just a single time step between each  update, but no matter how I change the drive cycle (averaging two time samples, omitting alternate samples, etc.), it seems that the process never proceeds further after "Start solver set-up" or while descritization. I have to terminate it manually.

What can be the possible reasons? I monitored some parameters like current, voltage, power, temperature, sei length, and concentration for each cell and time step, but nothing seems out of order.

Discretisation error:

![1E](https://user-images.githubusercontent.com/59028917/218732017-d4e77ce0-c5fb-416d-8ef7-e80f9aed6c10.png)

Solver error:

![x2](https://user-images.githubusercontent.com/59028917/218732381-c3ad4834-5cfe-4491-97d1-a413b57474e6.png)
![x3](https://user-images.githubusercontent.com/59028917/218732391-dbee28f2-ae18-4d9e-8e14-18e2d285a338.png)

**Comment by cyasing** [comment]
I use set_initial_conditions_from() for consecutive time steps.

**Reply by TomTranter** [reply]
@cyasing can you please stop posting liionpack discussions here. This isn't a discussion either it should be part of the issue already open on this topic

**Reply by cyasing** [reply]
Well, this is a different thing I was trying with just PyBaMM. Other than that, I will put it in issues.
